### PR TITLE
CI: setup automatic builds

### PR DESCRIPTION
**Pull request by ebolyen**
Note:

I haven't hooked up q2lint because I wasn't sure how we want to handle the copyright holder, and aside from that, the reference license hard-codes the year to `2016-2017`, so there's not a great way to keep the `2017` year for this particular license without changing q2lint slightly.

### CODE REVIEW

**Comment by ebolyen** [comment]
Also, tests won't pass until busywork flushes out a new environment file (that's expected).

File: q2_sample_classifier/__init__.py [hunk 1]
```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # ----------------------------------------------------------------------------
 # Copyright (c) 2017--, q2-sample-classifier development team.
```
Comment (thermokarst) [inline q2_sample_classifier/__init__.py]: Besides the date issue @ebolyen mentioned, should the Copyright notice name the `QIIME 2 development team`, like in other projects within @qiime2?

Comment (jairideout) [inline q2_sample_classifier/__init__.py]: :+1:, I think we should standardize on the copyright notice we're using in other repos (at least until there's a need to be more general).

**Comment by thermokarst** [comment]
Ignoring the drop in coverage --- that makes no sense to me, you made virtually no code changes. Thanks @ebolyen!

**Pull request merged**